<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Gevent on 647 Universe</title>
    <link>https://bwangel.me/tags/gevent/</link>
    <description>Recent content in Gevent on 647 Universe</description>
    <generator>Hugo</generator>
    <language>cn</language>
    <lastBuildDate>Mon, 06 Nov 2023 11:16:52 +0800</lastBuildDate>
    <atom:link href="https://bwangel.me/tags/gevent/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Gevent 的 KeyError</title>
      <link>https://bwangel.me/docs/python/gevent-keyerror/</link>
      <pubDate>Wed, 14 Sep 2016 15:21:40 +0000</pubDate>
      <guid>https://bwangel.me/docs/python/gevent-keyerror/</guid>
      <description>&lt;p&gt;&lt;strong&gt;摘要&lt;/strong&gt;:&lt;/p&gt;&#xA;&lt;blockquote&gt;&#xA;&lt;ol&gt;&#xA;&lt;li&gt;本文翻译自 StackOverFlow 上的一篇&lt;a href=&#34;http://stackoverflow.com/questions/8774958/keyerror-in-module-threading-after-a-successful-py-test-run&#34;&gt;答案&lt;/a&gt;&lt;/li&gt;&#xA;&lt;li&gt;本文主要解释了&lt;code&gt;gevent&lt;/code&gt;的猴子补丁和一个&lt;code&gt;KeyError&lt;/code&gt;之间的关系&lt;/li&gt;&#xA;&lt;/ol&gt;&lt;/blockquote&gt;</description>
    </item>
  </channel>
</rss>
